Server Backend/routes/users-route.js

```diff
@@ -1,8 +1,181 @@
 /**
  * Imports the `express` and `router` modules
  */
-const express = require('express')
-const router = express.Router()
+const express = require("express");
+const router = express.Router();
+
+/**
+ * Users array
+ */
+var users = [];
+
+/**
+ * Route that creates a user
+ */
+router.post("/createUser", function (req, res, next) {
+    /**
+     * TODO generate shortID and JWT
+     */
+
+    /**
+     * Create new user object
+     */
+     const newUser = {
+        userName: req.body.userName
+    }
+
+    /** 
+     * Add new user object to users array
+     */
+    users.push(newUser);
+    
+    /**
+     * Logs to the console what user was added
+     */
+    console.log("User: " + newUser.userName + " added.")
+
+    /**
+     * Returns to the user a JSON object containing the new user
+     */
+    res.json(newUser);
+});
+
+router.put("/updateScore", function (req, res, next) {
+    /**
+     * The word score to be updated
+     */
+    const wscore = req.body.user;
+
+    /**
+     * The player score to be updated
+     */
+    const wscore = req.body.user;
+    
+    /**
+     * TODO update scores on the database
+     */
+    
+
+    /**
+     * TODO Returns a success/fail boolean
+     */
+    
+});
+
+router.get("/getUser", function (req, res, next) {
+    /**
+     * The JWT of the user to be retreived
+     */
+    const userToken = req.body.user;
+
+    /**
+     * TODO look up the user on the databse
+     */
+
+    
+    /**
+     * TODO returns a JSON containing user information
+     */
+    res.json();
+});
+
+router.get("/currentTeam", function (req, res, next) {
+    /**
+     * The JWT of the user to be retreived
+     */
+    const userToken = req.body.user;
+
+    /**
+     * TODO look up the user on the databse
+     */
+
+    
+    /**
+     * TODO returns a String for the user's team
+     */
+    
+});
+
+router.put("/setTeam", function (req, res, next) {
```

Comment:
We will have GUI have a selection of which team the user wants to join and it will hit this endpoint.  Like it!